app/views/paths/show.html.erb

```diff
@@ -1,8 +1,14 @@
 <div class="container">
   <h1 class="text-center camel light curriculum-title"><%= @path.title %></h1>
+  <% if user_signed_in? %>
```

Comment:
I think it might read a little better if these conditionals were combined into one:
`<% if  user_signed_in? && current_user.path != @path %>` 

whats your thoughts?

Reply:
I think it's a great idea! I'll have an update in a couple seconds.